### ISSUE TITLE
Install fails

### ISSUE THREAD

**Issue by tescophil** [jeroenterheerdt/ring-hassio#22]
Hi,

Added this repo to the add-in store and hit the install button on the ring-hassio add in page.

Nothing, just spins its wheels for as long as you have patience for (I left it running for 10 mins..), no error messages in the logs or on screen.

Worse still, this totally bricked my HA install, everything stopped working, including those services working in other docker containers run by the HassOS supervisor. First time I've had to physically power off my system in 18 months...

arch | armv7l
-- | --
chassis | embedded
dev | false
docker | true
docker_version | 19.03.11
hassio | true
host_os | HassOS 4.12
installation_type | Home Assistant OS
os_name | Linux
os_version | 4.19.127-v7
python_version | 3.8.3
supervisor | 235
timezone | Europe/London
version | 0.114.3
virtualenv | false

**Comment by jeroenterheerdt** [comment]
not sure what happened here.

**Comment by ajnozari** [comment]
The repository.json is pointing to a different repo from the main jeroenterheerdt/ring-hassio.  Not sure if that would make a difference.

**Comment by jeroenterheerdt** [comment]
thanks, good catch. I'll revert that change. That could very well fix the issue.

**Comment by Brahmah** [comment]
Apologies, I seem to have left my repo in there when testing a pull!

**Comment by jeroenterheerdt** [comment]
Yeah, I tried to fix it, but that apparently did not help. I do not seem to receive the latest version myself so I can't really troubleshoot this.
